packages/polaris-viz/src/components/LineChart/components/PointsAndCrosshair/PointsAndCrosshair.tsx

```diff
@@ -70,6 +70,15 @@ export function PointsAndCrosshair({
     }
     const offset = isCrosshair ? selectedTheme.crossHair.width / 2 : 0;
 
+    const allSingleDataPoints = data.every(
```

Comment:
Do you think using `.some` instead so we're not always checking the entire data set would be better, or not worth the optimization?

```
const allSingleDataPoints = data.some(
  (series) => series.data.length !== 1,
);
```

Reply:
@envex now I'm a little confused by the logic of using `some` in this case. We need to check that every series in the data array has one value, like in this example:
```
const DATA= [
  {
    "name": "Apr 1 – Apr 14, 2020",
    "data": [
      {
        "value": 8,
        "key": "2020-04-02T12:00:00"
      },
    ]
  },
  {
    "name": "May 1 – May 14, 2020",
    "data": [
      {
        "value": 10,
        "key": "2020-04-02T12:00:00"
      },
    ]
  },
  {
    "name": "Jun 1 – Jun 14, 2020",
    "data": [
      {
        "value": 15,
        "key": "2020-04-02T12:00:00"
      },
    ]
  },
]
```
If only some of them have a single data point and some have several data points, we don't want to bypass the animation